repo2docker/buildpacks/base.py

```diff
@@ -100,7 +100,7 @@
 {% if build_script_files -%}
 # If scripts required during build are present, copy them
 {% for src, dst in build_script_files|dictsort %}
-COPY {{ src }} {{ dst }}
+COPY --chown=${NB_USER}:${NB_USER} {{ src }} {{ dst }}
```

Comment:
adding NB_USER as a template variable in addition to build arg will allow us to do `{{ NB_USER }}` here and support docker engine back to 17.09 instead of requiring 19.03.